content/TruenasApps/UsingTrains.md

```diff
@@ -29,20 +29,23 @@ Users can manage the catalog by changing the trains included in it.
 To add or remove a train, from the **Installed** application screen, click **Settings** then on **Train Settings** on the dropdown menu to open the **Train Settings** screen.
 You can also click **Manage Installed Apps** at the top right of the **Discover** screen to open the **Installed** screen.
 
-{{< trueimage src="/images/SCALE/Apps/AppsDiscoverScreen.png" alt="Applications Discover Screen" id="Applications Discover Screen" >}} new image
+{{< trueimage src="/images/SCALE/Apps/AppsDiscoverScreen.png" alt="Applications Discover Screen" id="Applications Discover Screen" >}}
 
-Users can change the catalog trains and show apps in those trains on the **Train Settings** screen. After making a change, go to the **Discover** screen and click on **Refresh Catalog** to pull the catalog from the repository and refresh it in SCALE.
+Users can change the catalog trains and show apps in those trains on the **Train Settings** screen.
+After making a change, go to the **Discover** screen and click on **Refresh Catalog** to pull the catalog from the repository and refresh it in SCALE.
 
 {{< trueimage src="/images/SCALE/Apps/AppsTrainSettingsScreen.png" alt="Apps Train Settings Screen" id="Apps Train Settings Screen" >}}
```

Comment:
This image is missing

Reply:
Fixed image, and changed content to the correct UI screen names.